projects/sb-user-management/src/lib/user-list/user-list.component.ts

```diff
@@ -1,15 +1,36 @@
-import { Component, OnInit } from '@angular/core';
-
+import { Component, OnInit } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
+import { UserService } from "../services/user-service/user.service";
 @Component({
-  selector: 'lib-user-list',
-  templateUrl: './user-list.component.html',
-  styleUrls: ['./user-list.component.css']
+  selector: "lib-user-list",
+  templateUrl: "./user-list.component.html",
+  styleUrls: ["./user-list.component.css"],
 })
 export class UserListComponent implements OnInit {
+  tableColumns: any[] = [];
+  users: any;
+  count: any;
+  constructor(private userService: UserService) {}
 
-  constructor() { }
+  ngOnInit() {
+    this.tableColumns = [
+      { field: "firstName", header: "First Name" },
```

Comment:
can we use i18 lang constants and files for UI labels?

Reply:
Yes, we can use it. Added this action item to the task list.